functions.php

```diff
@@ -69,13 +69,19 @@ function makePlantEntryTile(array $entry): string {
 }
 
 function getErrorMessage(string $error): string {
-    $error_string = ['1' => 'Please enter data for all fields.',
-                    '2' => 'Oops! Something went wrong. Please try again.',
-                    '3' => "The plant type given wasn't recognised. 
+    if ($error === '1' || $error === '2' || $error === '3' || $error === '4') {
```

Comment:
this isnt very maintainable as everytime you change the array ull have to change the if statement. instead use `array_key_exists`